3p/facebook.js

```diff
@@ -49,10 +49,16 @@ function getPostContainer(global, data) {
     setStyle(c, 'text-align', 'center');
   }
   const container = global.document.createElement('div');
-  const embedAs = data.embedAs || 'post';
+  let embedAs = data.embedAs || 'post';
   user().assert(['post', 'video'].indexOf(embedAs) !== -1,
       'Attribute data-embed-as  for <amp-facebook> value is wrong, should be' +
       ' "post" or "video" was: %s', embedAs);
+  // Force the `data-embed-as` attribute to 'video' and make sure to show the post's text.
+  if (data.href.indexOf('/videos/') > 0) {
```

Comment:
This would generate false positives if Facebook username has the word `videos` in it. Would need to tighten up the regex to only match `*/(videos)/d+`